dcat/config.js

```diff
@@ -327,6 +327,10 @@ var respecConfig = {
 			publisher:"Mind Informatics",
 			date:"16 March 2015"
         },
-        
+        "SPDX":{
```

Comment:
My first question is - should we check the governance and sustainability of SPDX? I see no changes since a few years back. @andrea-perego you may know more about its status and if it is used widely. Otherwise, I wonder if we should consider creating the terms in DCAT namespace (as it was the case for PAV).

Reply:
SPDX (https://spdx.dev/) is an open standard, so I don't think there are issues about governance and sustainability.

Reply:
thank you - the info I had found was https://wiki.spdx.org/view/SPDX_FAQ, which is older and didn't seem maintained, but the website that you mentioned (and associated github repo) show a different picture